README.md

```diff
@@ -309,12 +309,16 @@ fetch the logs of an app
 
 ```
 USAGE
-  $ liara app logs [-h] [--debug] [--api-token <value>] [--region iran|germany] [-a <value>] [-s <value>]
+  $ liara app logs [-h] [--debug] [--api-token <value>] [--region iran|germany] [-a <value>] [-s <value>] [-t]
+    [-f] [-c]
 
 FLAGS
   -a, --app=<value>    app id
+  -c, --colorize       Colorize log output
+  -f, --follow         Follow log output
   -h, --help           Show CLI help.
   -s, --since=<value>  show logs since timestamp
+  -t, --timestamp      Show timestamps
```

Comment:
Let's use the plural version: https://docs.docker.com/engine/reference/commandline/logs/#options

```suggestion
  -t, --timestamps      Show timestamps
```